.github/workflows/performance.yml

```
@@ -49,7 +49,7 @@ jobs:
 
             - name: Compare performance with trunk
```

Maybe we should change the name of the step
```suggestion
            - name: Compare performance with base branch
```

---

Yep, done.